packages/protocol/contracts/layer1/preconf/impl/PreconfRouter2.sol

```diff
@@ -95,15 +95,21 @@ contract PreconfRouter2 is EssentialContract, IProposeBatch {
                 (bytes32 registrationRoot, bytes memory data) =
                     abi.decode(nextLookaheadUpdateData, (bytes32, bytes));
                 lookaheadStore.updateLookahead(registrationRoot, data);
+
+                // Refetch the lookahead hash for the next epoch.
+                nextLookaheadHash = lookaheadStore.getLookaheadHash(nextEpochTimestamp);
```

Comment:
Maybe we don't need `(bytes32 registrationRoot, bytes memory data)` as `bytes32 registrationRoot` + `ILookaheadStore.LookaheadSlot[] memory nextLookahead` is sufficient enough for the registration?